code/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+  <title>Weather</title>
+</head>
+
+<body class="body" id="body">
+  <header>
+    <section id="headerPicture" class="weather-image"></section>
+    <section id="headerTemp" class="header-temp"></section>
+  </header>
+  <section class="main-container">
+    <section id="today" class="today">
+      <section id="weather" class="weather">
+        Loading...
+      </section>
+      <section id="SunRiseSet" class="sun-rise-set"></section>
```

Comment:
Your naming convention in other places begins with a small letter first, maybe this should be the same?
```suggestion
      <section id="sunRiseSet" class="sun-rise-set"></section>
```